frontend/src/Pages/SignupPage/SignupPage.jsx

```diff
@@ -0,0 +1,190 @@
+                    <InputLabel htmlFor='lastName' style={{ fontWeight: 'bold' }}>Last Name</InputLabel>
+                    <TextField
+                    id='lastName' label='Enter last name' variant='filled' name='lName' onChange={handleInput}
+                    value={userInfo.lName} error={!!error.lName} helperText={error.lName} InputProps={{ style: { width: 250 } }}/>
+                      </Grid>
+                  </Grid>
+                </Grid>
+                <Grid>
+                  <InputLabel htmlFor='username' style={{ fontWeight: 'bold' }}>Username</InputLabel>
+                  <TextField
+                    id='username' label='Enter username' variant='filled' name='username' onChange={handleInput}
+                    value={userInfo.username} error={!!error.username} helperText={error.username} InputProps={{ style: { width: 550 } }}/>
+                </Grid>
+                <Grid>
+                  <InputLabel htmlFor='email' style={{ fontWeight: 'bold' }}>Email</InputLabel>
+                  <TextField
+                    id='email' label='Enter email' variant='filled' name='email' onChange={handleInput}
+                    value={userInfo.email} error={!!error.email} helperText={error.email} InputProps={{ style: { width: 550 } }}
+                  />
+                </Grid>
+                <Grid>
+                  <InputLabel htmlFor='password' style={{ fontWeight: 'bold' }}>Password</InputLabel>
+                  <TextField
+                    id='password' type='password' label='Enter password' variant='filled' name='password' onChange={handleInput}
+                    value={userInfo.password} error={!!error.password} helperText={error.password} InputProps={{ style: { width: 550 } }}/>
+                </Grid>
+                <Grid>
+                  <InputLabel htmlFor='confirmpassword' style={{ fontWeight: 'bold' }}>Confirm Password</InputLabel>
+                  <TextField
+                    id='confirmpassword' type='password' label='Enter password' variant='filled' name='confirmPassword' onChange={handleInput}
+                    value={userInfo.confirmPassword} error={!!error.confirmPassword} helperText={error.confirmPassword} InputProps={{ style: { width: 550 } }}
+                  />
+                </Grid>
+                <Grid item style={{ display: 'flex', flexDirection: 'column', alignItems: 'flex-start', justifyContent: 'flex-end' }}>
+                  <InputLabel style={{ marginTop: 20 }}>
+                    <FormControlLabel
+                        control={<Checkbox checked={checked} onChange={isChecked} />}
+                        label={
+                            <Typography>
+                        I have read and accept <span style={{ fontWeight: 'bold' }}>Terms and Conditions</span>
+                      </Typography>}/>
+                  </InputLabel>
+                  <Button
+                    variant='contained' type='submit' style={{
```

Comment:
Please move inline styling to a separate css file for the sign up page; Overall, the structure and the use of Material UI is awesome but inline styling will make the file look a lot messier in the future as we add on to it! :) This applies for all other inline styling in the page as well

Reply:
fixed